documentation/manual/filters.md

```diff
@@ -24,34 +24,34 @@ The OSHDB is able to cope well even with complex polygons that have many vertice
+This is the most commonly used filter to select a certain subset of OSM data. It returns only OSM entities that match a given OSM tag. Elements can be filtered by either the presence of a tag [key](https://docs.ohsome.org/java/oshdb/0.6.0/aggregated/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.html#osmTag(java.lang.String)), the presence of an [exact tag](https://docs.ohsome.org/java/oshdb/0.6.0/aggregated/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.html#osmTag(java.lang.String,java.lang.String)), or other matching condition that tests the presence of specific tags or tag combinations.
 <!-- list and document all versions: collection(tag), key+collection(values), etc. -->
 
 osmEntityFilter
 ---------------
 
-It is possible to [define custom filtering functions](https://docs.ohsome.org/java/oshdb/0.6.0/aggregated/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.html#osmEntityFilter-org.heigit.bigspatialdata.oshdb.api.generic.function.SerializablePredicate-), that take an OSM entity object as input and decide wether each individual entity should be included in the result or not by returning a boolean value.
+It is possible to [define custom filtering functions](https://docs.ohsome.org/java/oshdb/0.6.0/aggregated/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.html#osmEntityFilter(org.heigit.bigspatialdata.oshdb.api.generic.function.SerializablePredicate)), that take an OSM entity object as input and decide wether each individual entity should be included in the result or not by returning a boolean value.
 
 _ohsome_ filter
 ---------------
 
-An easy way to provide complex [`filter`s](https://docs.ohsome.org/java/oshdb/0.6.0/aggregated/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.html#filter-org.heigit.bigspatialdata.oshdb.api.generic.function.String-) is through the functionality of [ohsome filters](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/libs/ohsome-filter#readme), which allow one to define osm data filter in a human readable syntax. With these one can combine several tag, type and geometry filters with arbitrary boolean operators.
+An easy way to provide complex [`filter`s](https://docs.ohsome.org/java/oshdb/0.6.0/aggregated/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.html#filter(java.lang.String)) is through the functionality of [ohsome filters](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/libs/ohsome-filter#readme), which allow one to define osm data filter in a human readable syntax. With these one can combine several tag, type and geometry filters with arbitrary boolean operators.
```

Comment:
note that this also fixes an unrelated bug (`org.heigit.bigspatialdata.oshdb.api.generic.function.String` :arrow_right: `java.lang.String`) in this javadoc link